NAMESPACE

```diff
@@ -12,3 +12,21 @@ export(summarySE)
 export(.plotPriorPosterior)
 export(.ttestBayesianGetBFnamePlots)
 export(.descriptivesPlotsRainCloudFill)
+export(.ttestReadData)
+export(.ttestCheckErrors)
+export(.ttestBayesianGetErrorsPerVariable)
+export(.confidenceLimitsEffectSizes)
+export(.ttestBayesianReadData)
```

Comment:
These can go because of the copying (except for `export(.confidenceLimitsEffectSizes)`)